src/proto/uprotocol/uuri.rs

```diff
@@ -12,44 +12,37 @@
  ********************************************************************************/
 
 use crate::uprotocol::uri::UUri as uproto_Uuri;
+use crate::uprotocol::SerializationError;
```

Comment:
Can we remove the alias for the `UUri` type? It doesn't seem to conflict with anything else, does it?